apps/client/src/pages/dashboard/utils/thumbnail.ts

```
@@ -0,0 +1,163 @@
+
+    for (const size of faviconSizes) {
+      const testUrl = getFaviconUrl(domain, size);
+      if (testUrl) {
+        try {
+          const response = await fetch(testUrl, { method: 'HEAD' });
+          if (response.ok) {
+            faviconUrl = testUrl;
+            break;
+          }
+        } catch {
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)